cmd/completion/main.go

```diff
@@ -8,16 +8,20 @@ import (
 	"github.com/k3s-io/k3s/pkg/cli/cmds"
 	"github.com/k3s-io/k3s/pkg/cli/completion"
 	"github.com/sirupsen/logrus"
-	"github.com/urfave/cli"
```

Comment:
We cannot use cli/v2 until containerd v2 is released (where they move from v1 to v2)